routes/candidates_routes.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const app = express();
+
+app.use(express.json());
+
+app.route("/candidates/:candidateId")
+    .get((req, res) => {
+        console.log("A GET request received")
+    })
+
+    .post((req, res) => {
+        console.log("A POST request received")
+    })
+
```

Comment:
We need to export the express router here, not the whole express app.
A node js application should contain only one express app with multiple routers.